js/controller.js

```diff
@@ -270,15 +270,11 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
       this.accessibilityControls = new AccessibilityControls(this); //keyboard support
       this.state.screenToShow = CONSTANTS.SCREEN.INITIAL_SCREEN;
-
-      if (this.videoVr) {
-        this.mb.subscribe(OO.EVENTS.VR_DIRECTION_CHANGED, 'customerUi', _.bind(this.setViewingDirection, this));
-      }
     },
 
-    setVideoVr: function(event, obj) {
-      this.videoVr = true;
-      this.videoVrSource = obj.source || null; //if we need video vr params
+    setVideoVR: function(event, obj) {
+      this.videoVR = true;
+      this.videoVRSource = obj.source || null; //if we need video vr params
```

Comment:
This is not super important, but I would recommend using a more descriptive name for `obj`. Also, do we need a null check for `obj` or is it guaranteed to exist?

Reply:
fix